src/app/cli/src/init/transaction_snark_profiler.ml

```diff
@@ -142,6 +142,10 @@ let pending_coinbase_stack_target (t : Transaction.t) stack =
 let profile (module T : Transaction_snark.S) sparse_ledger0
     (transitions : Transaction.t list) preeval =
   let constraint_constants = Genesis_constants.Constraint_constants.compiled in
+  let txn_global_slot =
```

Comment:
`state_body` is at top-level here. Could the global slot also be at top-level? It's computed several times, and I think it's always the same value.